lib/rb_lib_text/version.rb

```diff
@@ -1,3 +1,3 @@
 module RbLibText
-  VERSION = "0.0.1"
+  VERSION = "0.0.2"
```

Comment:
@johnnytomcat @sarahikeda sorry, is it possible to have like a SNAPSHOT version, or is that not a Ruby lib thing?